packages/nodes-base/test/nodes/Helpers.ts

```diff
@@ -1,5 +1,5 @@
 import { readFileSync, readdirSync, mkdtempSync } from 'fs';
-import { BinaryDataManager, Credentials, loadClassInIsolation } from 'n8n-core';
```

Comment:
`loadClassInIsolation` loads the compiled filed from `dist`, in `vm` context, which prevents jest from instrumenting those files for coverage. That's why for test, I've switched this over to using `require` on the source typescript files instead.